### DISCUSSION TITLE
Always use correct yeartext font

### DISCUSSION THREAD

**Discussion by mtdvlpr** [sircharlo/meeting-media-manager#813] [Ideas] [answered]
I found the cdn url for the Wt-ClearText-Bold font:
https://b.jw-cdn.org/fonts/wt-clear-text/1.019/Wt-ClearText-Bold.woff2

We could use this (either download it or reference this url) to always show the correct font instead of depending on users having installed Watchtower Library. What do you think, @sircharlo?

**Comment by sircharlo** [comment, marked as answer]
Ooh nice! I like it. 

I think that downloading the font into the local cache is the definitely they way to go. I always try to think of what the app is doing, multiplied by all the congregations in the world, to imagine how we can reduce server load and calls to APIs and such. 

I think it makes much more sense to download the font **once**, if the local cache size is different than the remote file (129kb at the moment), than to download it every time.

**Reply by mtdvlpr** [reply]
Alright, coming right up :)

**Reply by sircharlo** [reply]
You didn't happen to find the JW Icons font too?

**Reply by mtdvlpr** [reply]
See #814 :)
